src/library_exceptions.js

```diff
@@ -393,6 +395,7 @@ var LibraryExceptions = {
     var thrownType = info.get_type();
     var catchInfo = new CatchInfo();
     catchInfo.set_base_ptr(thrown);
+    catchInfo.set_adjusted_ptr(thrown);
```

Comment:
How are `catchInfo`'s base ptr and adjusted ptr different?

Reply:
I'm not sure exactly but this is features of libcxxabi.   I think it relates to virtual inheritance.